src/avp/address.rs

```diff
@@ -140,4 +146,14 @@ mod tests {
         let avp = Address::decode_from(&mut cursor, 18).unwrap();
         assert_eq!(avp.0.to_string(), "::1");
     }
+
+    #[test]
+    fn test_encode_decode_e164() {
+        let avp = Address::new(Value::E164(UTF8String::new("359898000135")));
+        let mut encoded = Vec::new();
+        avp.encode_to(&mut encoded).unwrap();
+        let mut cursor = Cursor::new(&encoded);
+        let avp = Address::decode_from(&mut cursor, 12).unwrap();
```

Comment:
ipv4 len is 6 (4 + 2)
ipv6 len is 18 (16 + 2)
e164 len here should then be (12 + 2) = 14